### PR TITLE
new animated button: aura pulse button

### PR DESCRIPTION
**Pull request by genarocolusso**
Issue: New animated button . I hope you guys like it.
you can rename it if you want.

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -69,6 +69,11 @@ $(document).ready(function () {
           label: "Fill Color - Up",
           classes: "fill-color-btn up-fill",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:72]: Place the button in the correct alphabetical order under "Animated"

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi thank you for contributing! Please next time create a "new button idea" issue first before working on a button.

Can you make the animation a little slower? I think it would look better that way.

**Comment by genarocolusso** [comment]
Sorry, I am doing it right now?

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @genarocolusso for code and idea

**Pull request merged**